packages/database-typeorm/__tests__/typeorm-sessions.ts

```diff
@@ -0,0 +1,280 @@
+import { randomBytes } from 'crypto';
+import { Connection, createConnection } from 'typeorm';
+import { AccountsTypeorm, entities } from '../src';
+
+const generateRandomToken = (length = 43): string => randomBytes(length).toString('hex');
+
+const user = {
+  username: 'johndoe',
+  email: 'john@doe.com',
+  password: 'toto',
+};
+
+describe('TypeormSessions', () => {
+  let userId: string;
+  let session: Record<string, string>;
+
+  let connection: Connection;
+  const url = 'postgres://postgres@localhost:5433/accounts-js-tests';
+
+  beforeAll(async () => {
+    // const adminConnection = await createConnection({
```

Comment:
can you please delete the commented code?

Reply:
sorry, my bad, will do